ser574-green-team/taigit
# US137-Task147: Get all the user story titles for analysis

To test the code: 
import {get_user_story_titles} from '../libraries/Taiga';

{console.log(get_user_story_titles('sanaydevi-ser-574').then((val)=> 
              {console.log('fe: ', val)}
              ))}

Output: ["Readable app", "LoC Counting", "(Optional) Taiga Linking", "Student Teams", "As a frontend developer, I want analysis on code c… show the relationship between the two to a user.", "As a professor I would like information about the …o that I can get a basic overview of the project.", "Code Complexity Metrics", "As a user, I want to see a detailed view of Github…ar understanding of a teams GitHub and Taiga data", "As a developer I would like the information taken … to make it easier for the UX team to display it.", "As a frontend developer, I want to create components work done by the Taiga team to appear on the UI", "As a frontend developer, I want to create componen… for the work done by the GitHub team into our UI", "As a user, I want the UI to respond and look consistent, so that I can navigate the site easily", "As a frontend developer, I want to add data taken …he Redux store so that the UI can access the data", "As a frontend developer, I want to add data taken … the Redux store, so that the UI can use the data", "As a code analyst, I want a documented selection o…can best select the tool to analyze repositories.", "As a backend developer, I want to know what data t…can give data to the front end team to visualize.", "As a front end developer, I want to have a functio…rieve data in order to visualize it for endusers.", "As a backend developer I want to test functions th…the data that will be sent to the front end team.", "As a backend developer for Taiga I want to better … more metrics for the front end team to work with", "As a backend developer for github I need access to…connect user stories, tasks, and commit messages.", "As a backend developer, I would like a well docume… can better understand what I am able to analyze.", "As a developer, I would like a standardize structu…rom a repository so that I know what to implement", "UI - As a user, I want to see a dashboard so that I can see a basic overview of a team project", "UI - As a developer, I want a deep understanding o…edux so that I can properly engineer this project", "As a user, I want to see an individual view of a t… can understand their contribution to the project", "As a developer I would like the information taken … to make it easier for the UX team to display it.", "As a backend developer I want to know what data th… that I can present it to the front end developer", "As a user, I want to have a login page where I can…ant to view so that I can choose specific project", "As a backend developer I want to test functions th…the data that will be sent to the front end team.", "As a backend developer for Taiga I want to better … more metrics for the front end team to work with"]
